woodwork/statistics_utils/_infer_temporal_frequencies.py

```diff
@@ -12,7 +14,8 @@ def _infer_temporal_frequencies(dataframe, temporal_columns=None):
         temporal_columns (list[str], optional): Columns for which frequencies should be inferred. Must be columns
             that are present in the DataFrame and are temporal in nature. Defaults to None. If not
             specified, all temporal columns will have their frequencies inferred.
-
+        debug (boolean): A flag used to determine if more information should be returned for each temporal column if
+                no uniform frequency was found.
     Returns:
         (dict): A dictionary where each key is a temporal column from the DataFrame, and the
```

Comment:
This should be updated to indicate that the return will be a tuple when debug=True!

Reply:
done